socorro/cron/crontabber_app.py

```diff
@@ -1013,16 +1014,18 @@ def _remember_success(
         app_name = class_.app_name
         execute_no_results(
             connection,
-            """INSERT INTO crontabber_log (
+            """INSERT INTO cron_log (
                 app_name,
                 success,
-                duration
+                duration,
+                log_time
             ) VALUES (
+                %s,
                 %s,
                 %s,
                 %s
             )""",
-            (app_name, success_date, '%.5f' % duration),
+            (app_name, success_date, '%.5f' % duration, utc_now()),
```

Comment:
Django `DateTimeField` has an `auto_add_now`, but it's implemented in Django-land and not db-land, so we need to add `utc_now()` here.